core/src/main/java/jenkins/model/ItemCategory/Category.java

```diff
@@ -0,0 +1,68 @@
+package jenkins.model.ItemCategory;
+
+import hudson.model.TopLevelItem;
+import org.kohsuke.stapler.export.Exported;
+import org.kohsuke.stapler.export.ExportedBean;
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Represents an {$link ItemCategory} and its {@link TopLevelItem}s.
+ */
+@ExportedBean
+public class Category implements Serializable {
+
+    private String id;
+
+    private String name;
+
+    private String description;
+
+    private String iconClassName;
+
+    private int weight;
+
+    private List<Map<String, Object>> items;
+
+    public Category(String id, String name, String description, String iconClassName, int weight, List<Map<String, Object>> items) {
```

Comment:
:ant: the interface allows to put the non-serializable object into <code>items</code>. Makes sense to replace Object by Serializable